bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,28 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='csscomb',
+        output_format='corrected',
+        use_stdout=True,
+        use_stdin=False,
+        result_message='Change coding style based on CSS rules written in ' +
+                       '.csscomb.json',
+        )
+class CSSCombBear:
+    """
+    CSScomb is a coding style formatter for CSS.
+
+    For more information, consult <https://github.com/csscomb/csscomb.js>.
+    """
+
+    LANGUAGES = {"CSS"}
+    REQUIREMENTS = {NpmRequirement('csscomb')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
```

Comment:
can we generate a config file with at least the most common settings? https://github.com/csscomb/csscomb.js/blob/dev/doc/configuration.md